student.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+    super(age, name)
```

Comment:
- [ ] parent_permission should be passed to super as well otherwise it will always be "true" even if the user enters false